### DISCUSSION TITLE
Nothing working

### DISCUSSION THREAD

**Discussion by Laamaaar** [piotrulos/MSCModLoader#195] [Help me]
Everything is done as the instructions say
![Снfwимок](https://github.com/piotrulos/MSCModLoader/assets/113025499/7649fd80-22ff-49e1-be6b-54608aca4171)
![Снимgregeок](https://github.com/piotrulos/MSCModLoader/assets/113025499/3a2edbd2-f83d-4eb5-a34d-8705d635d366)
![Снимwfgwgwок](https://github.com/piotrulos/MSCModLoader/assets/113025499/37049f18-61cc-4e6b-a28b-dfac3a5de387)

**Comment by piotrulos** [comment]
Looks like you skipped step in instructions.

**Reply by Laamaaar** [reply]
I have done everything as it says in the instructions, all packages are installed including unity engine and net framework 3.5

**Reply by piotrulos** [reply]
you need to install unity tools not unity engine.

**Reply by Laamaaar** [reply]
Yes i have it also

**Reply by Laamaaar** [reply]
Now I have installed everything as shown in the screenshots, nothing has changed at all

**Reply by Laamaaar** [reply]
The strangest thing is that visual studio doesn't even see MSCLoader

**Reply by piotrulos** [reply]
That happens when you don't read but look at screenshots, because the first paragraph has link to correct unity tools version

**Reply by Laamaaar** [reply]
You changed the page just 1 hour ago )

**Reply by Laamaaar** [reply]
Well, seems i fix problem with net framework 3.5
I also installed unity tools 2015 from the link, but nothing changed. I just ran the executable and it installed. Is there anything else I haven't done?
